tst/CTA.Rules.Test/Metrics/PortSolutionResultReportGeneratorTests.cs

```diff
@@ -246,30 +246,11 @@ public void GenerateAndExportReports_Creates_Expected_Json_Report()
     ""metricsType"": ""CTA"",
     ""metricName"": ""UpgradePackage"",
     ""packageName"": ""Newtonsoft.Json"",
-    ""packageVersion"": ""12.0.0.0"",
+    ""packageVersion"": ""12.0.0"",
+    ""packageOriginalVersion"": ""9.0.0"",
     ""solutionPath"": ""5fa9de0cb5af2d468dfb1702b1e342f47de2df9a195dabb3be2d04f9c2767482"",
     ""projectGuid"": ""1234-5678""
   },
-  {
-    ""metricsType"": ""CTA"",
-    ""metricName"": ""GenericAction"",
-    ""actionName"": ""GA1 Name"",
-    ""actionType"": ""GA1 Type"",
-    ""actionValue"": ""GA1 Value"",
-    ""solutionPath"": ""5fa9de0cb5af2d468dfb1702b1e342f47de2df9a195dabb3be2d04f9c2767482"",
-    ""projectGuid"": ""1234-5678"",
-    ""filePath"": ""eb98c1d648bc61064bdeaca9523a49e51bb3312f28f59376fb385e1569c77822""
-  },
-  {
-    ""metricsType"": ""CTA"",
-    ""metricName"": ""GenericAction"",
-    ""actionName"": ""GA2 Name"",
-    ""actionType"": ""GA2 Type"",
-    ""actionValue"": ""GA2 Value"",
-    ""solutionPath"": ""5fa9de0cb5af2d468dfb1702b1e342f47de2df9a195dabb3be2d04f9c2767482"",
-    ""projectGuid"": ""1234-5678"",
-    ""filePath"": ""eb98c1d648bc61064bdeaca9523a49e51bb3312f28f59376fb385e1569c77822""
-  },
```

Comment:
Should we have a unit test for the analysis metrics? It could help document the format of the metrics being logged.